### PR TITLE
Changes the "less" prompt to remind user to press q to exit.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: cmd/agree/agree.go [hunk 1]
```diff
@@ -222,6 +222,7 @@ func printTerms(ctx *cmd.Context, terms []terms.GetTermsResponse) error {
 	}
 	buffer := bytes.NewReader([]byte(output))
 	less := exec.Command("less")
+	less.Args = []string{`-P"Press 'q' to quit: stdin."`}
```
Comment (cmars) [inline cmd/agree/agree.go]: `less.Args = []string{"less", "-P", "Press 'q' to quit after you've read the terms."}`

**Comment by cmars** [comment]
Almost, see comment.

File: cmd/agree/agree.go (hunk 1 above)
Comment (alesstimec) [inline cmd/agree/agree.go]: done

**Comment by cmars** [comment]
$$merge$$

**Pull request merged**